demo/go/kv.go

```diff
@@ -61,6 +58,19 @@ func (kv *KV) Read(ctx context.Context, key string) (any, error) {
 	}
 }
 
+// ReadStruct reads the value of a key in the key/value store and store it in the value pointed by v.
+// Returns an *RPCError error with a KeyDoesNotExist code if the key does not exist.
+func (kv *KV) ReadStruct(ctx context.Context, key string, v any) error {
```

Comment:
wouldn't this be better as `ReadInto`? I don't see anything in here particularly limiting to a struct.

that way you can use this in both `ReadInt` (by passing an `&int`) as well as `Read` (by passing `&any`).``

Reply:
Thank you for your comment, yes, totally agree